build.gradle

```diff
@@ -41,9 +41,7 @@ dependencies {
     testRuntime 'org.objenesis:objenesis:2.6'
 }
 
-allprojects {
-    sourceCompatibility = 1.8
-}
+sourceCompatibility = 1.8
```

Comment:
Being explicit about target compatibility would make this easier to comprehend for me.

Reply:
Done.